backend/dataall/core/stacks/services/stack_service.py

```diff
@@ -213,3 +233,21 @@ def get_stack_logs(target_uri, target_type):
                     | filter @logStream like "{stack.EcsTaskArn.split('/')[-1]}"
                     """
         return query
+
+    # Adding target_type as an argument as this is needed by the function `map_target_type_to_log_config_path` when resolving path to config
+    @staticmethod
+    @is_feature_enabled_for_allowed_values(
+        allowed_values=['admin-only', 'enabled', 'disabled'],
+        enabled_values=['admin-only', 'enabled'],
+        default_value='enabled',
+        resolve_property=map_target_type_to_log_config_path,
+    )
+    def check_if_user_allowed_view_logs(target_type: str, target_uri: str, config: str):
+        context = get_context()
```

Comment:
nit: instead of passing the config as argument we can directly look it up inside the check_if_user_allowed_view_logs function.